Jenkinsfile

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu && immutable' }
+  environment {
+    BASE_DIR="src/github.com/elastic/package-storage"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
```

Comment:
@kuisathaverat Not sure if this is valid here?

Reply:
it is not used.

Reply:
removed